addons/sale_mrp/data/mail_template_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="exception_sale_on_manufacture_quantity_decreased" name="Message: Alert on sale orders when manufactured quantity decreased on Manufacturing Orders">
```

Comment:
Side guideline note (pinged by bot): mail_template_data is used for real email template (record of mail.template model). This kind of templates should generally be in mail_templates.xml (templates used in mail features, allowing to find them cross-modules but outside of real mail templates).

Cheers !